src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,35 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void setSize(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @Test
+    void contains() {
+        assertThat(numbers.contains(1)).isTrue();
+        assertThat(numbers.contains(2)).isTrue();
+        assertThat(numbers.contains(3)).isTrue();
```

Comment:
컬렉션을 쉽게 테스트할 수 있도록 메서드를 지원하고 있습니다.
```suggestion
        assertThat(numbers).contains(1);
        assertThat(numbers).contains(2);
        assertThat(numbers).contains(3);
```

Reply:
`@ParameterizedTest`와 `@ValueSource(ints = {1, 2, 3})`를 활용하면 중복을 쉽게 제거할 수 있어보여요!